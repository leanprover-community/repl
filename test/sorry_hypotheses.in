{"cmd": "theorem foo (x : Int) : x = x := by sorry"}

{"proofState" : 0, "tactic": "have h : x = 1 := sorry"}

