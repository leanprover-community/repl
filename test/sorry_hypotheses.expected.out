{"sorries":
 [{"proofState": 0,
   "pos": {"line": 1, "column": 36},
   "goal": "x : Int\n⊢ x = x",
   "endPos": {"line": 1, "column": 41}}],
 "messages":
 [{"severity": "warning",
   "pos": {"line": 1, "column": 8},
   "endPos": {"line": 1, "column": 11},
   "data": "declaration uses 'sorry'"}],
 "env": 0}

{"sorries": [{"proofState": 1, "goal": "x : Int\n⊢ x = 1"}],
 "proofState": 2,
 "goals": ["x : Int\nh : x = 1\n⊢ x = x"]}

